jms/src/main/scala/akka/stream/alpakka/jms/javadsl/JmsSink.scala

```diff
@@ -16,66 +16,40 @@ import scala.compat.java8.FutureConverters
 @deprecated("Use JmsProducer instead", "0.18")
 object JmsSink {
 
-  /**
-   * Java API: Creates an [[JmsSink]] for [[JmsMessage]]s
-   */
-  def flow[R <: JmsMessage](
-      jmsSinkSettings: JmsProducerSettings
-  ): akka.stream.javadsl.Flow[R, JmsMessage, NotUsed] =
-    akka.stream.alpakka.jms.scaladsl.JmsSink.flow(jmsSinkSettings).asJava
-
   /**
    * Java API: Creates an [[JmsSink]] for [[JmsMessage]]s
    */
   def create[R <: JmsMessage](
       jmsSinkSettings: JmsProducerSettings
   ): akka.stream.javadsl.Sink[R, CompletionStage[Done]] =
-    akka.stream.alpakka.jms.scaladsl.JmsSink
-      .apply(jmsSinkSettings)
-      .mapMaterializedValue(FutureConverters.toJava)
-      .asJava
+    JmsProducer.create(jmsSinkSettings)
```

Comment:
Rename the parameters as well, looks a bit silly that they just repeat the (old) class name, so maybe just call them `settings`?